sources/us/al/calhoun.json

```diff
@@ -17,6 +17,7 @@
     "conform": {
         "type": "geojson",
         "number": "STREET_NUM",
-        "street": "STREET_NAME"
+        "street": "STREET_NAME",
+        "postcode": "zipcode_1"
```

Comment:
Unfortunately, `zipcode_1` is for the owner mailing address, not the zip code for the situs.  The values vary wildly and correspond to the `STATE` field (which include many other states besides AL).